third_party/move/move-model/bytecode/src/stackless_control_flow_graph.rs

```diff
@@ -212,6 +254,12 @@ impl StacklessControlFlowGraph {
         self.blocks.keys().cloned().collect()
     }
 
+    /// Iterates over all blocks in DFS order starting from the entry block
```

Comment:
Please note that blocks not reachable from the entry block may be visited in arbitrary order.  (If visit_all is selected)

Reply:
Yes.